nunit.netstandard.sln

```diff
@@ -0,0 +1,43 @@
+
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.25123.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".SolutionItems", ".SolutionItems", "{76D1A814-CDD2-4A17-A983-49B54926BB0E}"
+	ProjectSection(SolutionItems) = preProject
+		.gitignore = .gitignore
+		global.json = global.json
+		README.md = README.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "NunitEngine", "NunitEngine", "{73808E53-CBA8-43AF-83DB-9D8E31AD2228}"
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "nunit.agent", "src\NetStandard\NunitEngine\nunit.agent.xproj", "{D1567C63-4A0D-4E18-A14E-79699B9BFFFF}"
```

Comment:
Our standard is to keep different targets for the same projects in the same directory. Since you put them under NetStandard, I assume you ran into the same problem I did with the `xproj` and `project.json` files being in the same directory as other `csproj` files?